docs-src/stargate-develop/modules/develop/examples/docapi/library/curl-document-patch-book-with-path.sh

```diff
@@ -0,0 +1,29 @@
+curl -X 'PATCH' \
+  'http://{base_doc_url}{base_doc_api}/{namespace}/collections/{collection}/{bookdocid}/book' \
+  -H 'accept: application/json' \
+  -H "X-Cassandra-Token: {auth_token}" \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "book": {
+    "title": "Native Daughter",
+    "isbn": "12322",
+    "author": [
+        "Richard Wright"
+    ],
+    "pub-year": 1930,
+    "genre": [
+        "slavery",
```

Comment:
Should we leave this in? From the perspective of inclusion, and since this is the "front page" of our API, I feel like at the very least this line should be omitted.

Reply:
changed to poverty. I actually thought it was about slavery, and there's nothing wrong with pointing that out.